mpmp/src/model/Player.java

```diff
@@ -6,368 +6,366 @@
 import java.util.Iterator;
 
 /**
- * Player represents spectators and the actual, active players. The info stored here exists in both
+ * Player represents spectators and the actual, actuve players. The info stored here exists in both
  * client and server and is kept up-to-date by the clientlist-update packet.
  *
  * @author Leander, oki
  */
 public class Player {
 
-	// XXX move to a new file of constants?
-	private static final int StartCash = 30000; // XXX value
-	private static final int Wage = 4000;  // XXX value
-	private static final int IncomeTax = 4000;  // XXX value
-	private static final int ExtraTax = 2000;  // XXX value
-	public static final int UnjailFee = 1000;
-	private static Player currentPlayer;
-
-	public enum Mode {
-
-		Spectator, Player
+    // XXX move to a new file of constants?
+    private static final int StartCash = 30000; // XXX value
```

Comment:
No, you can set it once in NetBeans. You might've changed your setting on the school comp, but forgot it to do it on your home box.

**EDITOR WARS INITIATED**
This can't happen to me. When I type a tab in `sam`, it is a tab. If I type four spaces, there are four spaces. If I end a line, the indentation from the previous line is not carried over, so I won't have lines with dangling whitespace in them. No automatic clse-brace generation exists. No syntax highlighting exists, the color scheme is superb. I do so love `sam` :heart: